plugin/db/pg/sync.go

```diff
@@ -305,10 +306,9 @@ func getPgTables(txn *sql.Tx) ([]*tableSchema, error) {
 	}
 
 	var tables []*tableSchema
-	query := "" +
-		"SELECT tbl.schemaname, tbl.tablename, tbl.tableowner, pg_table_size(c.oid), pg_indexes_size(c.oid) " +
-		"FROM pg_catalog.pg_tables tbl, pg_catalog.pg_class c " +
-		"WHERE schemaname NOT IN ('pg_catalog', 'information_schema') AND tbl.schemaname=c.relnamespace::regnamespace::text AND tbl.tablename = c.relname;"
+	query := "SELECT i.relid, tbl.schemaname, tbl.tablename, tbl.tableowner, pg_table_size(i.relid), pg_indexes_size(i.relid)" +
+		"FROM pg_catalog.pg_statio_user_tables AS i, pg_catalog.pg_tables AS tbl " +
```

Comment:
Using pg_statio_user_tables is not intuitive as that's for collecting io statistics.

For solving the double quotes comparison, we can use 

`quote_ident(tbl.schemaname)=c.relnamespace::regnamespace::text`

https://www.postgresql.org/docs/current/datatype-oid.html

Reply:
Or we should avoid join pg_class at all and use

pg_table_size((quote_ident(tbl.schemaname) || '.' || quote_ident(tbl.tablename))::regclass)

Reply:
QUOTE_IDENT(tbl.schemaname) = QUOTE_IDENT(c.relnamespace::regnamespace::text) not work. 
For schema created by using `CREATE SCHEMA "imp-1";`, QUOTE_IDENT(tbl.schemaname) returns `"imp-1"` but QUOTE_IDENT(c.relnamespace::regnamespace::text) returns `"""imp-1"""`.
<img width="443" alt="CleanShot 2022-08-24 at 10 16 25@2x" src="https://user-images.githubusercontent.com/87714218/186302844-da4b4ba2-667d-4e24-8134-9fef79da0e9b.png">
<img width="194" alt="CleanShot 2022-08-24 at 10 16 34@2x" src="https://user-images.githubusercontent.com/87714218/186302857-17f4c22c-51f3-41c8-b287-726406a0596e.png">

Reply:
But ` SELECT tbl.schemaname, tbl.tablename, pg_table_size((quote_ident(tbl.schemaname) || '.' || quote_ident(tbl.tablename))::regclass) FROM pg_catalog.pg_tables tbl;` works : )